dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -158,6 +155,64 @@ std::pair<UInt64, UInt64> MinMaxIndex::getUInt64MinMax(size_t pack_index)
     return {minmaxes->get64(pack_index * 2), minmaxes->get64(pack_index * 2 + 1)};
 }
 
+RSResult MinMaxIndex::checkNullableEqual(size_t pack_index, const Field & value, const DataTypePtr & type)
+{
+    const auto & column_nullable = static_cast<const ColumnNullable &>(*minmaxes);
+
+    const auto * raw_type = type.get();
+
+    // if minmaxes_data has null value, the value of minmaxes_data[i] is meaningless and maybe just some random value.
+    // But in checkEqual, we have checked the has_null_marks and ensured that there is no null value in MinMax Indexes.
```

Comment:
Move these comments to the front of the following lines is more clear
https://github.com/pingcap/tiflash/blob/97cef6476a13e335d0620bf77056a6ab5255548b/dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp#L224-L227